<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNodeEdgePropertiesUI</class>
 <widget class="QWidget" name="CNodeEdgePropertiesUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="NodesBox">
     <property name="title">
      <string>Nodes (1)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="4">
       <widget class="QGroupBox" name="NodeAttrBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Custom Attributes</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CAttributesEditorUI" name="NodeAttrEditor" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSint::QSplitButton" name="NodeShape">
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>32</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Set shape of the selected nodes</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="NodeSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>32</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Set size of the selected nodes</string>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QSint::ColorButton" name="NodeColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>44</width>
          <height>32</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Set fill color of the selected nodes</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QPushButton" name="NodeId">
        <property name="toolTip">
         <string>Click to edit</string>
        </property>
        <property name="statusTip">
         <string>Change Id of the selected node</string>
        </property>
        <property name="text">
         <string>Id: 123</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="EdgesBox">
     <property name="title">
      <string>Edges (1)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSint::ColorButton" name="EdgeColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>44</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set color of the selected edges</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSint::QPenButton" name="EdgeStyle">
          <property name="minimumSize">
           <size>
            <width>44</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set pen style of the selected edges</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="EdgeWeight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set weight of the selected edges</string>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSint::QSplitButton" name="EdgeDirection">
          <property name="minimumSize">
           <size>
            <width>44</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set direction of the selected edges</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="EdgeId">
        <property name="toolTip">
         <string>Click to edit</string>
        </property>
        <property name="statusTip">
         <string>Change Id of the selected edge</string>
        </property>
        <property name="text">
         <string>Id: E1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="EdgeAttrBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Custom Attributes</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CAttributesEditorUI" name="EdgeAttrEditor" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="LablesBox">
     <property name="title">
      <string>Labels</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSint::ColorButton" name="LabelColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>44</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set color of the selected labels</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSint::QFontButton" name="LabelFont">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set font of the selected labels</string>
          </property>
          <property name="text">
           <string>Aa1</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="LabelSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>Set size of the selected abels</string>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
          </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSint::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QSint::QSplitButton</class>
   <extends>QToolButton</extends>
   <header>qsplitbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QSint::QPenButton</class>
   <extends>QToolButton</extends>
   <header>qpenbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QSint::QFontButton</class>
   <extends>QToolButton</extends>
   <header>qfontbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CAttributesEditorUI</class>
   <extends>QWidget</extends>
   <header>CAttributesEditorUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>NodeColor</tabstop>
  <tabstop>NodeShape</tabstop>
  <tabstop>NodeSize</tabstop>
  <tabstop>NodeId</tabstop>
  <tabstop>NodeAttrBox</tabstop>
  <tabstop>EdgeColor</tabstop>
  <tabstop>EdgeStyle</tabstop>
  <tabstop>EdgeWeight</tabstop>
  <tabstop>EdgeDirection</tabstop>
  <tabstop>EdgeId</tabstop>
  <tabstop>EdgeAttrBox</tabstop>
  <tabstop>LabelColor</tabstop>
  <tabstop>LabelFont</tabstop>
  <tabstop>LabelSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>NodeAttrBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>NodeAttrEditor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EdgeAttrBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>EdgeAttrEditor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
